src/main/java/games/strategy/internal/persistence/serializable/VersionPersistenceDelegate.java

```diff
@@ -0,0 +1,22 @@
+package games.strategy.internal.persistence.serializable;
+
+import java.io.IOException;
+
+import games.strategy.persistence.serializable.AbstractPersistenceDelegate;
+import games.strategy.util.Version;
+
+/**
+ * A persistence delegate for the {@code Version} class.
+ *
+ * <p>
+ * Instances of this class are immutable.
+ * </p>
+ */
+public final class VersionPersistenceDelegate extends AbstractPersistenceDelegate {
```

Comment:
I think the only real need for having the `PersistenceDelegate` abstraction separate from the proxy is that the delegate is a container for the logic of the Serialization framework's four template methods:

* `annotateClass`
* `replaceObject`
* `resolveClass`
* `resolveObject`

It's unlikely that we will need the capability to customize `annotateClass`, `resolveClass`, and `resolveObject` from the persistence delegate.  So, yes, I agree, we can probably effectively move the logic for `replaceObject` to the proxies themselves.  As you pointed out, it will reduce `PersistenceDelegate` down to a functional interface.

Let me play around with that idea, and I'll see what I can come up with.